CombinatorialGames/Game/Loopy/Basic.lean

```diff
@@ -0,0 +1,371 @@
+/-
+Copyright (c) 2025 Aaron Liu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Liu, Violeta Hernández Palacios
+-/
+import CombinatorialGames.Game.Functor
+import CombinatorialGames.Mathlib.Small
+import Mathlib.Data.Setoid.Basic
+import Mathlib.Logic.Small.Set
+
+/-!
+# Loopy games
+
+The standard notion of a game studied in combinatorial game theory is that of a terminating game,
+meaning that there exists no infinite sequence of moves. Loopy games relax this condition, and thus
+allow "self-refential" games, with the basic examples being `on = {on | }`, `off = { | off}`, and
```

Comment:
There is a semantic difference here.

```suggestion
meaning that there exists no infinite sequence of moves. Loopy games relax this condition by
allowing "self-refential" games, with the basic examples being `on = {on | }`, `off = { | off}`, and
```

Reply:
Oh, I get what you mean now. Yeah, that's a better way to put it!